listenbrainz/listenstore/tests/test_timescalelistenstore.py

```diff
@@ -467,4 +451,12 @@ def test_000_delete_single_listen(self):
         min_ts, max_ts = self.logstore.get_timestamps_for_user(testuser_name)
         self.assertEqual(min_ts, 1400000000)
         self.assertEqual(max_ts, 1400000200)
-       
+
+    def test_for_empty_timestamps(self):
```

Comment:
add a docstring describing this test to remind us why we have a special case here:
`"""Even if a user has no listens they should have the sentinel timestamps of 0,0 stored in the cache to avoid continually recomputing them"""`
or similar